set bounding box to None to avoid layer cropping (#1908)
spacetelescope/jdaviz: CHANGES.rst

Imviz
^^^^^

> - Fixes cropped image layer with WCS-linnking without fast-approximation. [#1908]

Reviewer: ```suggestion
- Fixes cropped image layer with WCS-linking without fast-approximation. [#1908]
```

Imviz
^^^^^

> - Fixes cropped image layer with WCS-linking without fast-approximation, mouseover display

Reviewer: Not sure about hyphen but I should really defer to @ojustino et al.
```suggestion
- Fixes cropped image layer with WCS linking without fast-approximation, mouseover display
```

Reviewer 2: Since I was summoned, I'd say you're right about no hyphen on these, @pllim.

>   now shows when information is outside original reference data bounding box. [#1908]

Reviewer: ```suggestion
  for GWCS now shows when information is outside original reference data bounding box. [#1908]
```

>   for GWCS now shows when information is outside original reference data bounding box. [#1908]

Reviewer: ```suggestion
  for GWCS now shows when information is outside original bounding box, if applicable. [#1908]
```
